modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/OrganizationResourceTest.java

```diff
@@ -856,6 +863,64 @@ private Organization _addOrganization(
 		return organizationResource.postOrganization(organization);
 	}
 
+	private OrganizationContactInformation
```

Comment:
please call this _get*** because in this method you are not adding it anywhere, you are just returning it

Reply:
or probably better _getRandomOrganizationContactInformation